### PR TITLE
Feature/dashboard api

### PR DESCRIPTION
**Pull request by annashamray**
depends on https://github.com/GemeenteUtrecht/bptl/pull/33

### CODE REVIEW

File: src/bptl/dashboard/views.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        item_data = {q["status"]: q["tasks"] for q in item_qs}
+        items.append({type: item_data})
+
+    total_qs = BaseTask.objects.annotate_status()
+    total_data = {q["status"]: q["tasks"] for q in total_qs}
+
+    return {"items": items, "total": total_data}
+
+
+class AggregateView(APIView):
```
Comment (sergei-maertens) [inline src/bptl/dashboard/views.py]: I don't like mixing API endpoints and non-API views/urls. Can you put the API stuff in an `api/` package?

File: src/bptl/urls.py [hunk 2]
```diff
@@ -37,7 +37,8 @@
     ),
     # Simply show the master template.
     path("", TemplateView.as_view(template_name="index.html"), name="index"),
-    path("tasks/", include("bptl.tasks.urls")),
+    path("tasks/", include("bptl.dashboard.urls")),
```
Comment (sergei-maertens) [inline src/bptl/urls.py]: Please separate API endpoints from regular HTML views/URLs :)

File: src/bptl/dashboard/views.py (hunk 1 above)
Comment (annashamray) [reply src/bptl/dashboard/views.py]: `AggregateView` is moved to `dashboard/api` package

File: src/bptl/urls.py (hunk 2 above)
Comment (annashamray) [reply src/bptl/urls.py]: done

**Pull request merged**